package.json

```diff
@@ -18,6 +18,7 @@
     "@use-gesture/vanilla": "^10.2.27",
     "buffer": "^6.0.3",
     "cbor-x": "^1.5.4",
+    "clerk-sveltekit": "^0.4.1",
```

Comment:
let's remove this, we  won't use clerk here

Reply:
https://dev.to/danawoodman/how-to-secure-pages-with-http-basic-auth-using-sveltekit-1iod